.github/workflows/e2e.yml

```diff
@@ -28,10 +28,9 @@ jobs:
       - uses: pnpm/action-setup@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
We still support 18, so it was intentional. I don't mind changing to 20 though for e2e.